Components/input/input.css

```diff
@@ -0,0 +1,38 @@
+.input{
+    padding: .8rem;
+    border: transparent;
+    width: max-content;
+    min-width: 8rem;
+    background-color: var(--lynx-grey-color);
+    border: .1rem solid transparent;

```

Comment:
Use px unit for the values <= 10px. Because you dont want responsiveness for <=10 px

Reply:
Thanks !! updated in px units.